ui/package.json

```diff
@@ -19,9 +19,11 @@
     "rc-checkbox": "^2.1.8",
     "rc-time-picker": "^3.7.2",
     "react": "^16.9.0",
+    "react-checkbox-tree": "^1.6.0",
     "react-datepicker": "^2.10.1",
     "react-dom": "^16.9.0",
     "react-focus-lock": "^2.2.1",
+    "react-icons": "^3.9.0",
```

Comment:
Can we use the icon directly from font-awesome?

Reply:
Updated with font awesome icons.